File: api/models/entities/Asteroid.cs
namespace Orbital.Api.Models.Entities;

public class Asteroid
{
    public int Id { get; set; }
    public string NeoReferenceId { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public string NasaJplUrl { get; set; } = string.Empty;
    public decimal AbsoluteMagnitudeH { get; set; }
    public decimal EstimatedDiameterMinKm { get; set; }
    public decimal EstimatedDiameterMaxKm { get; set; }
    public bool IsPotentiallyHazardous { get; set; }
    public bool IsSentryObject { get; set; }
}

File: web/types/asteroid.ts
export interface Asteroid {
    neoReferenceId: string,
    name: string,
    nasaJplUrl: string,
    absoluteMagnitudeH: number,
    estimatedDiameterMinKm: number,
    estimatedDiameterMaxKm: number,
    isPotentiallyHazardous: boolean,
    isSentryObject: boolean,
    closeApproachDate: string,
    relativeVelocityKph: number,
    missDistanceKm: number
}